### PR TITLE
640 Reset your notification settings

### PR DESCRIPTION
**Pull request by Fabiomorais87**
Hi @jorge-campo,

In accordance with our agreement, I am sending the PR after we have completed the two steps.

1 - Meet with Jorge to determine if the article provides the user with all the information they need.

2 - Worked with Emma to polish the English.

I would appreciate any suggestions you have. :)

Fabio

### CODE REVIEW

**Review by jorge-campo — CHANGES_REQUESTED** [review]
Thanks for this content, @Fabiomorais87 – Here are just some minor changes I propose you to include:

- Intro. We can simplify the wording a bit. I used WordTune for this. Same number of words, but the wording is easier: 'If you set custom notification settings before, but you don't receive the notifications you expect anymore, you can reset your settings. Doing so removes any exceptions, and the notification settings return to their default values.'
- Procedure – You're missing the last step, where the user confirms they want to reset the notifications. Whenever you do these types of 'danger' actions (reset, delete, clear, etc.), it's almost certain you need to confirm the action. See [this Figma file](https://www.figma.com/file/JlpPhJp0SMnEyBQy4nOZHo/Settings-for-Mobile?type=design&node-id=4312-388322&mode=design&t=SdtLYcdM3ZTw2cau-4).

**Comment by jorge-campo** [comment]
Hi @Fabiomorais87 , what's the status of this? Could you let me know if you implemented the review comments? Thanks.

**Comment by Fabiomorais87** [comment]
This article has already been reviewed by campo and the content is good for publication. I already sent it to .mdx syntax review here https://github.com/status-im/status-website/pull/586. Waiting for the pr to be approved and then merge the content.
